pkg/orm/task.go

```diff
@@ -83,27 +83,6 @@ func (o *TaskORM) GetById(ctx context.Context, taskId string) (*db.TaskModel, er
 
 // Modified GetTasksByWorkerSubscription with caching
 func (o *TaskORM) GetTasksByWorkerSubscription(ctx context.Context, workerId string, offset, limit int, sortQuery db.TaskOrderByParam, taskTypes []db.TaskType) ([]db.TaskModel, int, error) {
-	// Convert TaskTypes to strings
-	typeStrs := make([]string, len(taskTypes))
-	for i, t := range taskTypes {
-		typeStrs[i] = string(t)
-	}
-	cacheKey := cache.BuildCacheKey(cache.TasksByWorker, workerId, strconv.Itoa(offset), strconv.Itoa(limit), strings.Join(typeStrs, ","))
-
-	var tasks []db.TaskModel
-	cache := cache.GetCacheInstance()
-
-	// Try to get from cache first
-	if err := cache.GetCacheValue(cacheKey, &tasks); err == nil {
-		totalTasks, err := o.countTasksByWorkerSubscription(ctx, taskTypes, nil)
-		if err != nil {
-			log.Error().Err(err).Msgf("Error fetching total tasks for worker ID %v", workerId)
-			return tasks, 0, err
-		}
-		return tasks, totalTasks, nil
-	}
-
-	// Cache miss, proceed with database query
```

Comment:
sorry i might've forgotten this, but why do we remove this from the cache?

Reply:
yep, unfortunately I have to remove this cache because of the pagination page doesn't updated properly after some changes occur on the task page.